provider/datagen/src/lib.rs

```diff
@@ -225,6 +225,9 @@ pub enum FallbackMode {
     /// they contain the same value, only "en" will be included, since "en-US" falls back to
     /// "en" at runtime.
     ///
+    /// By default, locales that inherit directly from "und" will be retained even if their data
+    /// is equal to "und". To override this behavior, see [`BaseLanguageHandling`].
+    ///
```

Comment:
issue: I'm not convinced this is a good default. This increases baked data size, but supported locale queries are not a universal use case.

Reply:
Happy to discuss the default. Given that it's needed for 402 and we've seen enough examples of use cases where people may want to do these queries out of the box, and the data size increase is fairly small (no regional variants), I lean toward including these by default. It's easy for power users to get the extra 5% data size benefit if they know what they are doing.

Reply:
slightly lean towards the default  of Strip. I think either way we will need careful documentation of supported locale queries and how they can work. We should do that documentation and clearly document the defaults.